lib/Model/EventTree.js

```diff
@@ -0,0 +1,114 @@
+module.exports = EventTree;
+
+function EventTree(parent, segment) {
```

Comment:
`EventListenerTree` would a name that more clearly indicated what this tree is holding.